homeassistant/components/kaleidescape/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""The Kaleidescape integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, NamedTuple
+
+from kaleidescape import Device as KaleidescapeDevice, KaleidescapeError
+
+from homeassistant.const import CONF_HOST, EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.exceptions import ConfigEntryNotReady, HomeAssistantError
+
+from .const import DOMAIN
+
+if TYPE_CHECKING:
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import Event, HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.MEDIA_PLAYER, Platform.REMOTE]
+
```

Comment:
It looks like typing is complete or mostly complete. You should add this integration to `.strict-typing` in the base of the repo

Reply:
Done. Resulted in catching a handful of typing issues. All fixed.